rules-java-archives/addon/pom.xml

```diff
@@ -58,10 +62,50 @@
             <classifier>forge-addon</classifier>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.jboss.windup.maven</groupId>
+            <artifactId>nexus-indexer-data</artifactId>
+            <version>${version.nexus.index}</version>
+            <type>zip</type>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 
     <build>
+        <resources>
+            <resource>
+                <directory>src/main/java</directory>
```

Comment:
I think the idea was to remove this resources section and go back to resources in the standard place (src/main/resources).